pkg/apis/machine/v1alpha1/types.go

```diff
@@ -992,6 +993,15 @@ type AzureOSDisk struct {
 	CreateOption string                     `json:"createOption,omitempty"`
 }
 
+type AzureDataDisk struct {
+	Name         string                     `json:"name,omitempty"`
+	Lun          int32                      `json:"lun,omitempty"`
+	Caching      string                     `json:"caching,omitempty"`
+	ManagedDisk  AzureManagedDiskParameters `json:"managedDisk,omitempty"`
```

Comment:
Can we make the `ManagedDisk` attribute flat? We want to specify only the volume type here? So why not simply renaming it `volumeType` and make it of type string or better `compute.StorageAccountTypes`.

Reply:
@guydaichs , this has been taken care of, right ?